admin_frontend/app/transaction/page.tsx

```
@@ -0,0 +1,471 @@
+    if (dateFilter !== 'all') {
+      const now = new Date();
+      const today = new Date(now.getFullYear(), now.getMonth(), now.getDate());
+      
+      filtered = filtered.filter(t => {
+        const transDate = new Date(t.createdAt);
+        
+        switch (dateFilter) {
+          case 'today':
+            return transDate >= today;
+          case 'week':
+            const weekAgo = new Date(today);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)